CMakeLists.txt

```diff
@@ -145,11 +145,12 @@ cmake_dependent_option(ENABLE_PYTHON_COVERAGE "Enable Python tests with coverage
 find_package(OpenSCAP REQUIRED)
 
 if (SSG_TARGET_OVAL_MAJOR_VERSION EQUAL "5" AND SSG_TARGET_OVAL_VERSION_MINOR EQUAL "11" AND NOT "${OSCAP_V_OUTPUT}" MATCHES "OVAL Version: 5.11")
-    message(FATAL_ERROR "Your version of OpenSCAP does not support OVAL 5.11, please switch the OVAL target version to 5.10 or lower. $ cmake -DSSG_TARGET_OVAL_MINOR_VERSION=10 ../")
+    message(FATAL_ERROR "Your version of OpenSCAP does not support OVAL 5.11, please upgrade to a newer version of OpenSCAP.")
```

Comment:
Maybe we can go even further and remove this check, because the `"OVAL Version: 5.11"` has been introduced in [OpenSCAP 1.2.2](https://github.com/Openscap/openscap/commit/0aa1a0d02046f26935aa0f3833e7edf531662237) so the error happens on RHEL 7.1 and older and I don't think anybody uses these old systems for content development.